scripts/jenkins/Makefile

```diff
@@ -6,13 +6,11 @@
 
 tests = $(shell ls -d */ | cut -f1 -d'/')
 
-REG?=intel/
-ORG?=$(REG:/=)
 TAG?=devel
 
 set-version:
-ifneq ($(ORG)$(TAG),inteldevel)
-	@/bin/bash ../set-version.sh --tag $(TAG) --org $(ORG)
+ifneq ($(TAG),devel)
+	@cd ../.. && ./scripts/set-version.sh $(TAG)
 endif
```

Comment:
set-version is useful in the top-level makefile: 
```
diff --git a/Makefile b/Makefile
index ebd09e3..36136ee 100644
--- a/Makefile
+++ b/Makefile
@@ -80,4 +80,7 @@ lock-images:
        @scripts/update-clear-linux-base.sh clearlinux/golang:latest $(shell ls build/docker/*.Dockerfile)
        @scripts/update-clear-linux-base.sh clearlinux:latest $(shell find demo -name Dockerfile)

-.PHONY: all format vet cyclomatic-check test lint build images $(cmds) $(images) lock-images vendor pre-pull
+set-version:
+       @scripts/set-version.sh $(TAG)
+
+.PHONY: all format vet cyclomatic-check test lint build images $(cmds) $(images) lock-images vendor pre-pull set-version
```

it's OK to have a separate 'set tag' stage before testing.

Reply:
done :)